assets/webpack.config.js

```diff
@@ -98,12 +98,23 @@ module.exports = (env, options) => ({
   module: {
```

Comment:
The only reason I changed this file is because I was running into problems with react components in tests and originally thought maybe jest was working from webpack-compiled code or something and went through line-by-line to understand what was going on. there aren't any functional changes here, I just merged some of the matchers.